deep-learning/src/test/scala/com/microsoft/azure/synapse/ml/onnx/ONNXModelSuite.scala

```diff
@@ -258,6 +259,13 @@ class ONNXModelSuite extends TestBase
       .setMiniBatchSize(1)
   }
 
+  // We must clear the softMax and argMax dictionaries, as they only apply to the full model transform
```

Comment:
should we make OnnxModelSuite use OnnxHub?

Reply:
you can if you want!

Reply:
ok, actually it makes sense to test pathways other than OnnxHub since we still support them, so I will leave it